### ISSUE TITLE
detailed payout info is not shown with 100% beneficiary

### ISSUE THREAD

**Issue by economicstudio** [steemit/condenser#3450]
#### Expected behavior

Detailed payout info should still be shown when a post with 100% beneficiary is paid out.

#### Actual behavior

Detailed payout info is not shown when a post with 100% beneficiary is paid out.

#### How to reproduce

![](https://user-images.githubusercontent.com/38183982/62007651-16cff580-b148-11e9-8f9c-852930c44f1d.png)

> https://steemit.com/@blockchainstudio/finex-steem-power-liquify
- There is no button to show detailed payout info

#### Cause and Fix
This is due a wrong nonzero payout check condition for paid out post. I'll fix this and submit PR.

Basically this will be more meaningful if beneficiary info is shown (which is done by me #3310), but that PR is still pending for a long time. Once you it is merged I'll also added it for paid out posts.

* Browser/App version: Any
* Operating system: Any

**Comment by anpigon** [comment]
요런 버그가 있었군요. 숨겨진 버그나 불편한 기능을 잘 찾으십니다. ㅋ
